lms/djangoapps/courseware/views/views.py

```diff
@@ -734,6 +735,16 @@ def create_page_context(self, request, course=None, tab=None, **kwargs):
             'openedx.course_experience.reset_course_deadlines', kwargs={'course_id': text_type(course.id)}
         )
 
+        display_reset_dates_banner = False
+
+        if RELATIVE_DATES_FLAG.is_enabled(course.id):
+            course_overview = CourseOverview.objects.filter(id=course.id).first()
```

Comment:
`CourseOverview` has a method to get the overview (and load it from the modulestore if it's not already in the database), `get_from_id`. I'd recommend using that, to insulate yourself from any future issues in case the overview gets updated.